website/docs/components/form/file-input/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: File Input
+description: A file input that enables users to select one or more files from their local device for upload.
+caption: A file input that enables users to upload files.
+links:
+  figma: {link to the "Form::FileInput" page in the components Figma library}
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/form/file-input
+related: ['path/to/component', 'path/to/pattern']
```

Comment:
[blocker] this needs to be updated (or removed)

Reply:
@andgen404 Could you please add related items here or else delete this line?